### ISSUE TITLE
Something fishy with the restAPI (endpoints)

### ISSUE THREAD

**Issue by rwidmark** [ironmansoftware/powershell-universal#3838]
### Version

5.0.8

### Severity

Low

### Environment

msi

### Steps to Reproduce

I have not change anything from 4.4.0 but still when I'm visiting the swagger for my restAPI it just say No operations defined in spec! (see screenshot below).
I can call all of my endpoints and it works, every singel one of them.

from endpoints.ps1
````
$Parameters = @{
    Url            = "/rest/Computer/Create/Computer"
    Method         = @('POST')
    Path           = "Computer/Create/Computer.ps1"
    Authentication = $true
    Role           = @('Administrator', 'RestAPI-Dashboard')
    ErrorAction    = "SilentlyContinue"
    Environment    = "Standard"
    Tag            = @('Computer')
    Documentation  = "MyEndpoint"
}
New-PSUEndpoint @Parameters
```` 

from endpointDocumentation.ps1
```` 
$Parameters = @{
    Name           = "MyEndpoint"
    Description    = "Documentation for MyEndpoint API"
    Authentication = $true
    Role           = @('RestAPI-Dashboard', 'Administrator')
    Definition     = {
        [Documentation()]
        class InPutGenerateComputerName {
            [string]$Prefix = "xxx, yyy or bbb"
        }
   }
    Url            = "/rest/MyEndpoint"
}
New-PSUEndpointDocumentation @Parameters
```` 

### Expected behavior

```console
that Endpoint documentation should show up.
```

### Actual behavior

```console
Don't show up, when i'm looking at it it looks like this.
{
  "openapi": "3.0.1",
  "info": {
    "title": "MyEndPoint",
    "description": "Documentation for MyEndPoint API"
  },
  "paths": {
    "/rest/Computer/Status/Skrot/{ComputerName}": { },
    "/rest/Computer/Status/Saknad/{ComputerName}": { },
```

### Screenshots/Animations

![CleanShot 2024-09-28 at 21 00 19@2x](https://github.com/user-attachments/assets/dae7f715-d1ba-4950-b0ce-f053ac42c1a4)

**Comment by rwidmark** [comment]
@adamdriscoll ping.

**Comment by adamdriscoll** [comment]
I can't reproduce this on some basic docs I've created. All our automated tests are passing on this too so we must be missing something. I'm sure it has something to do with a more complex example. Do you have any errors in the logs?

**Comment by rwidmark** [comment]
@adamdriscoll I can reproduce it now.
In endpoints.ps1 if you change the endpoint path to:
```
/test/test.ps1
```
then it will show in the documentation. But as a result it will delete everything in the test.ps1 file instead. and the folder test and also the file will be created inside of C:\
If I have the path like:
```
test/test.ps1
```
it will not overwrite the test.json file but then it will not show up in documentation 

I might be dumb or something but I never get this thing with the path changes for restAPI since v5 I have had issues with it all the time and this seems like one of them issues.

Edit:
I have tried the following path now in the endpoints.ps1 and it seem to work for now, it¨s bedtime now but here is the path that I did put in:
```
/ProgramData/UniversalAutomation/Repository/RestAPI/test/test.ps1
```
